### PR TITLE
Task s4 countdown

### PR DESCRIPTION
**Pull request by karimkefi**
Merge avatars movements and countdown into Story4

### CODE REVIEW

File: js/avatarMovement.js [hunk 1]
```diff
@@ -1,76 +1,49 @@
...
-var playerBx = 550;
-var playerBy = 550;
-var playerBHeight = 50;
-var playerBWidth = 50;
-var playerBOrientation = 'W';
-
-//create players
-var playerA = drawPlayer(playerAx, playerAy, playerAHeight, playerAWidth, 'pink', 'playerA', playerAOrientation);
-var playerB = drawPlayer(playerBx, playerBy, playerBHeight, playerBWidth, 'blue', 'playerB', playerBOrientation);
+var countdownStart = 3;
```
Comment (obwansan) [inline js/avatarMovement.js:1]: maybe put these 2 variables under // gameplay variables?

File: js/avatarMovement.js [hunk 2]
```diff
@@ -1,76 +1,49 @@
...
-    rect.setAttributeNS(null, 'y', y);
-    rect.setAttributeNS(null, 'height', h);
-    rect.setAttributeNS(null, 'width', w);
-    rect.setAttributeNS(null, 'fill', fill);
-    rect.setAttributeNS(null, 'id', id);
-    rect.setAttributeNS(null, 'orientation', orientation);
-    document.getElementById('svgOne').appendChild(rect);
-    return document.getElementById(id);
-}
+var $playerA = $('#playerA');
```
Comment (obwansan) [inline js/avatarMovement.js:8]: do you want the player variables to start with a $?

File: js/avatarMovement.js [hunk 3]
```diff
@@ -125,10 +98,38 @@ function moveDown($player) {
...
+            $("#timerDiv").toggle();
+            startGame = true;
+            console.log('at end of timer' + startGame);
+            $(document).trigger('myCustomEvent');
+        }
+    }, 1000)
+}
+
+function playGame() {
+    console.log('you can play');
```
Comment (obwansan) [inline js/avatarMovement.js:121]: need to remove console.log?

File: js/avatarMovement.js [hunk 4]
```diff
@@ -125,10 +98,38 @@ function moveDown($player) {
...
-document.addEventListener("keyup", keyUpHandler);
\ No newline at end of file
+$(document).on('myCustomEvent', function () {
+    playGame();
+    $(document).off("keypress");
+});
+
+$(document).on("keypress", function (){
+        myCountdown();
+        console.log('wait for countdown');
```
Comment (obwansan) [inline js/avatarMovement.js:133]: do you want to keep these console.logs?

**Review by obwansan — COMMENTED** [review]
a few minor comments

File: js/avatarMovement.js (hunk 1 above)
Comment (kyam-mayden) [reply js/avatarMovement.js:1]: resolved

File: js/avatarMovement.js (hunk 2 above)
Comment (kyam-mayden) [reply js/avatarMovement.js:8]: this is here to clarify that we using JQuery for these variables

File: js/avatarMovement.js (hunk 3 above)
Comment (kyam-mayden) [reply js/avatarMovement.js:121]: resvolved

File: js/avatarMovement.js (hunk 4 above)
Comment (kyam-mayden) [reply js/avatarMovement.js:133]: resolved

**Pull request merged**